src/main/webapp/components/filters/property-filter.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('managementConsole.filters', [])
+    .filter('customPropertyFilter', function (){
```

Comment:
Also, it would be nice to write some documentation here! This filter is important because it can filter deep properties, so that should be stated, and argument should be explained.